packages/backend/src/nest/qss/qss.service.ts

```diff
@@ -410,39 +410,55 @@ export class QSSService extends EventEmitter implements OnModuleDestroy, OnModul
    * @param dataSyncMessage Pending message we want to send to QSS
    * @returns True if sent successfully, false if send failed, and undefined if the send was skipped
    */
-  private async _sendDataSyncMessage(
-    dataSyncMessage: QSSDataSyncMessage,
+  private async _sendLogEntrySyncMessage(
+    dataSyncMessage: QSSLogEntrySyncMessage,
     address: string
   ): Promise<boolean | undefined> {
     const hash = dataSyncMessage.payload!.hash
     const teamId = dataSyncMessage.payload?.teamId
     if (!this.connected) {
       this.logger.warn('QSS not connected, writing entry to dead letter queue', hash, teamId)
-      await this.localDbService.addPendingQssSyncMessage(address, hash)
+      try {
+        await this.localDbService.addPendingQssLogSyncMessage(address, hash)
+      } catch (e) {
+        this.logger.error('Failed to write pending QSS log sync message to local DB', e)
+      }
+      return undefined
+    }
+
+    if (this.joinStatus(teamId) !== JoinStatus.JOINED) {
```

Comment:
It wasn't causing problems but quiet was sending sync data to qss without being signed in and was polluting logs with errors